test/app/src/routes.js

```diff
@@ -100,7 +100,10 @@ if (!urlParams.has('routemap')) {
         // This route has a static prop that is passed to it
         '/foo': wrap({
             component: Foo,
-            props: {staticProp: 'this is static'}
+            props: {
+                staticProp: 'this is static',
+                dynamicProp: async () => Promise.resolve(`this is dynamic - ${new Date}`),
```

Comment:
I think this example is a bit more convoluted than needed, as using promises is confusing to a lot of people.

Can you use an example with a function that uses `await`?